presto-spi/src/main/java/com/facebook/presto/spi/function/FunctionNamespaceManager.java

```diff
@@ -74,7 +74,7 @@
      * List all functions managed by the {@link FunctionNamespaceManager}.
```

Comment:
It would be nice if we could list the param in java doc. Especially it would be a best-effort pushdown